Header header

# положение в глобальной системе координат
float64 lon
float64 lat

# положение в относительной системе координат
float64 x
float64 y